certified-connectors/JotForm Enterprise/readme.md

```diff
@@ -0,0 +1,18 @@
+
+## JotForm Enterprise
+Create powerful forms and surveys and embed them in your website with JotForm. Build custom workflows and connect additional apps to your Enterprise account.
+
+## Pre-requisites
+You will need the following to proceed:
+ - A Microsoft PowerApps or Microsoft Flow plan with custom connector feature
+ - [JotForm Enterprise](https://www.jotform.com/enterprise/) Account
+
+## Supported Operations
+**When a response is submitted** - Triggers a flow when there is a new form submission. 
+
+## How to get credentials
+Please use [these instructions](https://www.jotform.com/help/253-How-to-Create-a-JotForm-API-key) to create a JotForm API Key.
+
+
+## Deployment instructions
+Please use [these instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/paconn-cli) to deploy this connector as custom connector in Microsoft Flow and PowerApps
```

Comment:
Please change this to "...as a custom connector in Microsoft Power Automate and Power Apps."

Reply:
Thanks for addressing!